app/config.py

```diff
@@ -86,14 +89,16 @@
     "WEBHOOK_SERVICE_HOSTNAME", "http://renku-graph-webhooks-service"
 )
 
-KEYCLOAK_URL = os.environ.get("KEYCLOAK_URL")
-if not KEYCLOAK_URL:
+KEYCLOAK_URL = os.environ.get("KEYCLOAK_URL", HOST_NAME.strip("/") + "/auth")
+if KEYCLOAK_URL:
+    KEYCLOAK_URL = KEYCLOAK_URL.strip("/")
```

Comment:
Doesn't this always test `True` since you define a default for `KEYCLOAK_URL` in the previous line?